2025 Team 09

MohawkIsaacE opened the pull request.

MohawkRaphaelT commented:
@Varsha-vk05 @hajihajhaj Are you looking for me to close your [previous PR](https://github.com/MohawkRaphaelT/mini-game-collection/pull/4)?

MohawkIsaacE committed:
Add team folders and scene template

MohawkIsaacE committed:
Copy example prefabs to our project

LostPhantom18 committed:
d

MohawkIsaacE committed:
Add player movement

MohawkIsaacE committed:
Fix player names

LostPhantom18 committed:
Merge branch 'main' into Jonah's-Branch

LostPhantom18 committed:
Update Team09.unity

MohawkIsaacE committed:
Add test object

LostPhantom18 committed:
Merge branch 'main' into Jonah's-Branch

MohawkIsaacE committed:
Add test object

LostPhantom18 committed:
Update Team09.unity

LostPhantom18 committed:
Merge branch 'main' into Jonah's-Branch

LostPhantom18 committed:
d

MohawkIsaacE committed:
Add test

LostPhantom18 committed:
Update Team09.unity

LostPhantom18 committed:
Merge branch 'main' of https://github.com/MohawkIsaacE/mini-game-collection

LostPhantom18 committed:
d

MohawkIsaacE committed:
test

LostPhantom18 committed:
Merge branch 'main' into Jonah's-Branch

LostPhantom18 committed:
Revert "Merge branch 'main' into Jonah's-Branch"

This reverts commit 5f65aafa446d79c40f474f97aefb78f1cfcbc88a, reversing
changes made to 298672c1bb217850b2a25ed16166cf85094c3a51.

WyattAiden committed:
create map 1

WyattAiden committed:
merge remote-tracking branch 'origin/main' into Wyatt

WyattAiden committed:
pulled from main

LostPhantom18 committed:
Added Player Score

LostPhantom18 committed:
Fixed Errors, Updated Team 09 Project to look correct.

We had an error where event system and arcade input wasnt setup correctly. It has been formatted to look like Raphs game and now everything works.

LostPhantom18 committed:
Added bomb throwing, Added lineup line to show aiming

- Added bomb throwing
- Added a line to show where you are aiming
- Speed pickups
- Speed pickups functionality
- Silver and Gold coin spawn over time

LostPhantom18 committed:
Update Team09.unity

LostPhantom18 committed:
d

LostPhantom18 committed:
Merge branch 'Jonah's-Branch'

LostPhantom18 committed:
Merge branch 'main' into Wyatt

LostPhantom18 committed:
Imported Wyatts Map

- Imported Wyatts Map
- Setup a border
- Added powerup spawing prevents on colliders
- Max spawn for certain items

LostPhantom18 committed:
Merge branch 'Jonah's-Branch'

LostPhantom18 committed:
Start screen, ready up, spawn fix.

- Added a start screen
- Added a ready up system
- Fixed spawning right away

LostPhantom18 committed:
Merge branch 'Jonah's-Branch'

LostPhantom18 committed:
Updated Start Text

MohawkIsaacE committed:
Fix scene name and movement

- Fixed movement directions
- Fixed direction of looking for movement
- Corrected the scene name

LostPhantom18 committed:
Fix naming for team09 and game scene

- Remove Intro from TEAM09 Folder
- Rename game scene
- Remove game objects that werent needed

MohawkIsaacE committed:
Update naming issues

- GiveReward.cs namespace
- Removed Intro.unity

LostPhantom18 committed:
Random Change

Random Change

LostPhantom18 committed:
Merge branch 'Jonah's-Branch'

MohawkIsaacE committed:
Merge branch 'main' into isaac

WyattAiden committed:
Merge remote-tracking branch 'origin/main' into Wyatt

WyattAiden committed:
pull from main

MohawkIsaacE committed:
Stop spawning items when game ends

LostPhantom18 committed:
Added Controls, Disable movement on start

- Added Controls
- Disabled movement on spawn
- Fix UI to be mirrored

WyattAiden committed:
add map #2

LostPhantom18 committed:
Merge branch 'main' into Jonah's-Branch

MohawkIsaacE committed:
Merge branch 'main' into isaac

MohawkIsaacE committed:
Add random map choosing

MohawkIsaacE committed:
Add gdd file

michaelburt-mohawk commented:
To fix before merging:
- Revert changes to `unity-project/mini-game-collection/ProjectSettings/EditorBuildSettings.asset`
- `Mini Game UI.prefab` file cannot go in the root folder, has to either go in your Team's folder or delete it if you don't need
- Resources should not go inside prefab folder, if you need, you can add a Resources/ folder to your team folder
- Need to follow prefab naming convention: `2025-team09-player.prefab`
- Same for scenes - follow naming convention
- If `BulletOwner.cs` is not needed, delete
- Remove `NamespaceExample.cs`

MohawkIsaacE committed:
Update Arena Arcade Collector.md

- Add table of contents
- Clean up formatting
- Adjust some wording to better reflect our results

MohawkIsaacE committed:
Fix pull request conflicts

- Removed unused scripts
- Added resources folder
- Renamed prefabs and scenes
- Reverted changes to MiniGameUI.prefab and EditorBuildSettings.asset
- Also fixed bomb shooting for player 2

MohawkIsaacE committed:
Revert ScoreKeeper.cs to older version

MohawkIsaacE committed:
Update Arena Arcade Collector.md

MohawkIsaacE committed:
Fix file names

Files include:
- GDD
- Hole.png

MohawkIsaacE commented:
@michaelburt-mohawk We have fixed what was listed.

michaelburt-mohawk commented:
Game merged successfully and is playable on the `development2025` branch, great work
